README.md

```diff
@@ -89,28 +96,40 @@ add this file to your project.
 ```
 
 ### Fetching translations
-As soon as `fetchTranslations` is called, the SDK will attempt to download the translations for all locales
-that are defined in the initialization of `TxNative`. 
 
-For the moment, the translations are stored only in memory and are available for the current app session.
-In later versions of the SDK, the translations will also be stored on the device and will be available for subsequent
-app sessions.
+As soon as `fetchTranslations` is called, the SDK will attempt to download the 
+translations for all locales that are defined in the initialization of `TxNative`. 
```

Comment:
Since we are at it, it would probably be good to specify that we are only talking about the target locales (not the source locale).

Reply:
Done!